x-pack/dev-tools/jest/setup/setup_test.js

```diff
@@ -11,3 +11,9 @@
 
 import 'jest-styled-components';
 import '@testing-library/jest-dom/extend-expect';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import * as testingLibraryDom from '@testing-library/dom';
+
+testingLibraryDom.configure({
+  testIdAttribute: 'data-test-subj',
```

Comment:
I got a bunch of type errors due to `data-testid` not being declared on EUI elements. Our component library shouldn't depend on 3rd party libraries naming conventions, so I reconfigured `@testing-librar' to use Kibana specific `data-test-subj` attribute instead.